proto/tink/bbs.proto

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright Google Inc., SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Definitions for BBS+.
+syntax = "proto3";
+
+package google.crypto.tink;
+
+option java_package = "com.google.crypto.tink.proto";
+option java_multiple_files = true;
+option objc_class_prefix = "TINKPB";
+option go_package = "github.com/hyperledger/aries-framework-go/crypto/tinkcrypto/primitive/proto/bbs_go_proto";
+
+// Protos keys for BBS+ signatures/proofs.
+//
+//
+// BBS-keys represent BBSSign, BSSVerify, BBSProof and BBSVerifyProof primitives.
+
+// BBSCurveType, bls12-381 is the only supported BBS+ curve.
+enum BBSCurveType {
+  UNKNOWN_KEY_TYPE = 0;
+  BLS12_381 = 1;
```

Comment:
This is not sufficient - how do you know its Bls12381G2

Reply:
I will add GroupField to the public key..

Reply:
added